client/blocks/reader-post-options-menu/index.jsx

```diff
@@ -115,30 +113,30 @@ const ReaderPostOptionsMenu = React.createClass( {
 					{ this.props.showFollow && <FollowButton tagName={ PopoverMenuItem } siteUrl={ followUrl } /> }
 
 					{ isEditPossible ? <PopoverMenuItem onClick={ this.editPost } icon="pencil">
-						{ this.translate( 'Edit Post' ) }
+						{ this.props.translate( 'Edit Post' ) }
 					</PopoverMenuItem> : null }
 
 					{ ( this.props.showFollow || isEditPossible ) && ( isBlockPossible || isDiscoverPost ) &&
 						<hr className="reader-post-options-menu__hr" /> }
-					{ isBlockPossible ? <PopoverMenuItem onClick={ this.blockSite }>{ this.translate( 'Block Site' ) }</PopoverMenuItem> : null }
-					{ isBlockPossible || isDiscoverPost ? <PopoverMenuItem onClick={ this.reportPost }>{ this.translate( 'Report this Post' ) }</PopoverMenuItem> : null }
+					{ isBlockPossible ? <PopoverMenuItem onClick={ this.blockSite }>{ this.props.translate( 'Block Site' ) }</PopoverMenuItem> : null }
+					{ isBlockPossible || isDiscoverPost ? <PopoverMenuItem onClick={ this.reportPost }>{ this.props.translate( 'Report this Post' ) }</PopoverMenuItem> : null }
 				</EllipsisMenu>
 			</span>
```

Comment:
I think we need `QueryReaderSite` in here in case we haven't got the site.

I notice we're including `QueryReaderFeed` up top.